internal/service/service_test.go

```diff
@@ -148,3 +148,142 @@ func TestServiceV2(t *testing.T) {
 		})
 	}
 }
+
+func TestServiceV2WithClusterMapping(t *testing.T) {
```

Comment:
I am missing some CI and pre release versions in this test

Reply:
PTAL: https://github.com/RedHatInsights/insights-operator-gathering-conditions-service/pull/266/commits/cddf1e76944c266b55ccd0b988e56710c27f578e